modules/slate-sugar/src/main/ts/ephox/slate-sugar/api/PredicateFilter.ts

```diff
@@ -0,0 +1,9 @@
+import { Maybes, Maybe } from '@ephox/katamari';
+import { ModelApi, SlateLoc } from './Api';
+
+const descendants = (api: ModelApi, scope: SlateLoc, predicate: (loc: SlateLoc) => boolean): Maybe<SlateLoc[]> =>
```

Comment:
If you look at Sugar `PredicateFilter` vs `PredicateFind`, `*Find` modules are intended to return a maybe single element; `*Filter` methods must always return an array (even if it's empty). This is true in the RTC methods you're binding to, as well, so the `Maybe<>` type here is unnecessary.

Reply:
I've removed the PredicateFilter.descendants result to be wrapped in Maybe